web/packages/teleterm/src/services/config/getKeyboardShortcutSchema.ts

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright 2023 Gravitational, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { z } from 'zod';
+
+import { Platform } from 'teleterm/mainProcess/types';
+
+const VALID_SHORTCUT_MESSAGE =
+  'Valid shortcut can contain multiple modifiers (like "Shift") and a single key code (like "A" or "Tab"), combined by the "+" character.';
```

Comment:
Should we accept shortcuts that consist of a single A-Z key code?

I just set my `keymap.newTab` to `T` and the moment I press `T` while filtering connections, logging in to a cluster etc. the handler captures the press and executes the shortcut action.

Reply:
I think we should not block this as it will complicate our validation rules. 
I don't see much benefit in that additional rule - the user can easily see what happened and change the shortcut.

Reply:
> the user can easily see what happened and change the shortcut

I feel like this is true only if the user intentionally sets the keybind to a single character. If you make a mistake and save the file with the modifier missing, I bet it's going to be difficult to pinpoint the issue.

Imagine I set `keymap.tab8` to `"8"` by mistake. If I don't open more than 8 tabs then I won't see the effects of the keybind, Connect will just eat 8 as an input. Which will be even harder to see if a password you use for Teleport or for some service you're ssh into includes this character.

How would it complicate the rules though? Wouldn't it be enough to check that the value includes at least one modifier or F1-F24? As in, we want to keep support for setting F1-F24 alone and for the rest of ALLOWED_KEY_CODES at least one modifier must be present.

Reply:
>Wouldn't it be enough to check that the value includes at least one modifier or F1-F24?

Hmm okay, it should be enough. I will add it!